Battleship/static.cs

```diff
@@ -0,0 +1,15 @@
+namespace Battleship
+{
+    public class Static
```

Comment:
```suggestion
    public static class Constants
```

Static is a modifier to class type. Not the name for it. A static class is one that does NOT require instantiation, i.e., it is not an object that can be created. They are also values loaded at start time, meaning they are available at the START of the program.